### PR TITLE
Adding farsi_news dataset (https://github.com/sci2lab/Farsi-datasets)

### PR DESCRIPTION
**Pull request by Narsil**

### CODE REVIEW

File: datasets/farsi_news/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
```
Comment (lhoestq) [inline datasets/farsi_news/README.md:20]: do you mind adding the rest of the dataset card ?

you can leave each paragraph with `[More Information Needed]`

File: datasets/farsi_news/farsi_news.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+"""
+
+_URL = "https://raw.githubusercontent.com/sci2lab/Farsi-datasets/master/farsi_news/"
+_URLS = {
+    "hamshahri": _URL + "hamshahri.json",
+    "radiofarda": _URL + "radiofarda.json",
+}
+
+
+class Medal(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/farsi_news/farsi_news.py:39]:
```suggestion
class FarsiNews(datasets.GeneratorBasedBuilder):
```

File: datasets/farsi_news/farsi_news.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import json
+
+import datasets
+
+
+_CITATION = """\
+"""
+
+_DESCRIPTION = """\
+
```
Comment (lhoestq) [inline datasets/farsi_news/farsi_news.py:29]: can you add a few lines describing the dataset here ?

and also add the citation if there's one ?

**Review by lhoestq — COMMENTED** [review]
Looks very good !

I left a few comment

File: datasets/farsi_news/farsi_news.py (hunk 3 above)
Comment (Narsil) [reply datasets/farsi_news/farsi_news.py:29]: There's no citation (only a github link, what should I do ?)

File: datasets/farsi_news/README.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+- YAML tags: copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/ or https://github.com/huggingface/datasets-tagging for local tagging
```
Comment (lhoestq) [inline datasets/farsi_news/README.md:2]:
```suggestion
```

File: datasets/farsi_news/farsi_news.py (hunk 3 above)
Comment (lhoestq) [reply datasets/farsi_news/farsi_news.py:31]: let's keep it empty then

**Review by lhoestq — APPROVED** [review]
Nice !

**Pull request merged**